QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,195 @@
+This document is a declaration of software quality for the `libstatistics_collector` package, based on the guidelines in [REP-2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst).

```

Comment:
The REP-2004 link is broken, expected?

Reply:
This is expected, eventually the rep-2004 PR will be merged...

Reply:
I think we should use the published link: https://www.ros.org/reps/rep-2004.html